OracleInstantClient/dockerfiles/18/Dockerfile

```diff
@@ -1,11 +1,8 @@
 # LICENSE UPL 1.0
 #
-# Copyright (c) 2014, 2019, Oracle and/or its affiliates. All rights reserved.
+# Copyright (c) 2014, 2020, Oracle and/or its affiliates. All rights reserved.
```

Comment:
If you're changing copyright notices anyway, please remove the "All rights reserved." phrase as well. We no longer include that phrase.